src/model/channel/channel_id.rs

```diff
@@ -137,9 +141,10 @@ impl ChannelId {
     /// Returns [`ModelError::BulkDeleteAmount`] if an attempt was made to
     /// delete either 0 or more than 100 messages.
     ///
-    /// [`Channel::delete_messages`]: enum.Channel.html#method.delete_messages
-    /// [`ModelError::BulkDeleteAmount`]: ../enum.ModelError.html#variant.BulkDeleteAmount
-    /// [Manage Messages]: permissions/constant.MANAGE_MESSAGES.html
+    /// [`Channel::delete_messages`]: ../channel/enum.Channel.html#method.delete_messages
+    /// [`ModelError::BulkDeleteAmount`]: ../error/enum.Error.html#variant.BulkDeleteAmount
```

Comment:
I am unsure about if I also should change this to [`Error::...`] But I think that makes most sense but it should be possible to do with a simple grep command

Reply:
let's specifically link to `ModelError`, since otherwise it's a bit unnecessary since all errors are known to be encapsulated in `serenity::Result<T, serenity::Error>` anyway.